coordinator/persistence/aggregation/src/integrationTest/kotlin/net/consensys/zkevm/persistence/dao/aggregation/AggregationsPostgresDaoTest.kt

```diff
@@ -469,6 +469,65 @@ class AggregationsPostgresDaoTest : CleanDbTestSuiteParallel() {
     }
   }
 
+  @Test
+  fun findHighestConsecutiveEndBlockNumber_returns_highest_consecutive_end_block_number_from_various_block_numbers() {
+    // insert aggregation of blocks 1..20 in db
+    val aggregation1 = Aggregation(
+      startBlockNumber = 1UL,
+      endBlockNumber = 20UL,
+      batchCount = 10UL,
+      aggregationProof = null // proof can be omitted here as not the focus of the test
+    )
+    aggregationsPostgresDaoImpl.saveNewAggregation(aggregation1).get()
```

Comment:
Minor: We're copypasting this creation and insertion for the same purpose multiple times. IMO it would be nice to reflect the similarity of their purpose by factoring them out into a function.